public/content/sv/welcome/links-documentation.md

```diff
@@ -0,0 +1,17 @@
+# Länkar och dokumentation
+
+Efter att ha avslutat rundturen, kan du kolla in några av de följande länkarna för mer information:
+
+**NOTERA:** Följande länkar är på engelska
```

Comment:
FYI: if there are any reference(s) in Swedish , just add them
(however I assume that there isn't much Swedish content)